csharp/src/Google.Protobuf/Reflection/EnumDescriptor.cs

```diff
@@ -128,12 +128,20 @@ public EnumValueDescriptor FindValueByName(string name)
         /// <summary>
         /// The (possibly empty) set of custom options for this enum.
         /// </summary>
-        [Obsolete("CustomOptions are obsolete. Use GetOption")]
+        [Obsolete("CustomOptions are obsolete. Use the Options property.")]
         public CustomOptions CustomOptions => new CustomOptions(Proto.Options?._extensions?.ValuesByNumber);
 
+        /// <summary>
```

Comment:
Document in what scenario that null will be returned?

Reply:
Improved the docs.